### PR TITLE
fix: Add some software-terms words that start with "un"

### PR DESCRIPTION
**Pull request by ssbarnea**
# Add/Fix Dictionary

Dictionary: software-terms

## Description

- add missing software-terms
- lowercase few terms
- correct sorting
- add correction for instantiatable

- https://english.stackexchange.com/questions/337696/what-is-the-proper-word-for-something-that-can-be-instantiated

## Checklist

- [x] By submitting this pull-request, you agree to follow our [Code of Conduct](https://github.com/streetsidesoftware/cspell-dicts/blob/main/CODE_OF_CONDUCT.md)
- [x] Verify that the title starts with the correct prefix:
  - `fix:` - for minor changes like adding words or fixing spelling issues.
  - `feat:` - for a significant change like adding a whole new set of words to a dictionary.
  - `feat!:` - for breaking changes, like file format or licensing changes.
  - `chore:` - for changes that do not impact the content of dictionaries.

Fixes: #3129
Fixes: https://github.com/streetsidesoftware/cspell/issues/5286
Needs: #3617

### CODE REVIEW

**Comment by ssbarnea** [comment]
That things with "4 workflows awaiting approval" does not seem very welcoming,... and I even contributed previously. Maybe a tune in settings would be possible?

**Comment by Jason3S** [comment]
> @Jason3S That things with "4 workflows awaiting approval" does not seem very welcoming,... and I even contributed previously. Maybe a tune in settings would be possible? ;)

I would be happy to add you as a contributor. 

The default security settings have gotten stricter over time.

File: dictionaries/en_US/src/additional_words.txt [hunk 1]
```diff
@@ -56,12 +56,31 @@ rebrand
 rebranding
 teleconverter
 tricep
+unaggregated
```
Comment (Jason3S) [inline dictionaries/en_US/src/additional_words.txt:59]: Please add these words (all the ones you added) to [coding-terms](https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/software-terms/src/coding-terms.txt).

This dictionary is for US English words, general computing/coding terms are better of in the software terms dictionary. That way, those that do not use US English will also get the benefit.

**Review by Jason3S — CHANGES_REQUESTED** [review]
@ssbarnea,

Thank you for the additions and taking the time to do it.

**Comment by Jason3S** [comment]
@ssbarnea,

It is ok to just add words to the end of the file, it will get sorted automatically.

**Comment by Jason3S** [comment]
@ssbarnea,

Would you like some help with this PR?

**Comment by ssbarnea** [comment]
@Jason3S no, is just that I got sidetracked by other activities. I just seen the invite from you a minute ago but it expired.

**Comment by Jason3S** [comment]
@ssbarnea,

> I just seen the invite from you a minute ago but it expired.

I'll send it again.

**Comment by ssbarnea** [comment]
@Jason3S Something unexpected happened when I updated this PR. I seen failing to run because pnpm lock was out of sync with the package.json, and I did not touch those files.

See https://github.com/streetsidesoftware/cspell-dicts/actions/runs/11125090808/job/30912037317

I have experience using reqs and lock files with npm and yarn, but not pnpm but I assume that they would work very similarly. The only case where this would happen is if someone would push a change bypassing the GHA pipelines, which I doubt is the case.

**Comment by Jason3S** [comment]
> @Jason3S Something unexpected happened when I updated this PR. I seen failing to run because pnpm lock was out of sync with the package.json, and I did not touch those files.
> 
> See https://github.com/streetsidesoftware/cspell-dicts/actions/runs/11125090808/job/30912037317
> 
> I have experience using reqs and lock files with npm and yarn, but not pnpm but I assume that they would work very similarly. The only case where this would happen is if someone would push a change bypassing the GHA pipelines, which I doubt is the case.
> 
> Maybe there is something incorrectly configured on GHA that allowed this to happen, or some caching went nuts?
> 
> As the issue reproduced locally, I raised #3617 which should address this issue.

Thank you. I hadn't realize the build was broken. The last publish PR had a stale lock file, causing it to fail. #3612 had an updated package.json, but not an updated pnpm-lock.yaml.

**Review by Jason3S — APPROVED** [review]

**Comment by Jason3S** [comment]
@ssbarnea,

Thank you for fixing the files.

**Pull request merged**